src/ess.jl

```diff
@@ -7,16 +7,10 @@ abstract type AbstractESSMethod end
 The `ESSMethod` uses a standard algorithm for estimating the
 effective sample size of MCMC chains.
 
-It is is based on the discussion by Vehtari et al. and uses the
-biased estimator of the autocovariance, as discussed by Geyer.
+It is is based on the discussion by [^VehtariGelman2021] and uses the
+biased estimator of the autocovariance, as discussed by [^Geyer1992].
```

Comment:
IIRC the reason I removed the duplicates was because we use `makedocs` with `strict=true`, which errors when there are duplicates. We'll need to change it to `strict=false`.